dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04305UpdateWorkflowActionTable.java

```diff
@@ -0,0 +1,633 @@
+    private static final String POSTGRES_SELECT_SCHEME_IDS_FOR_ACTIONS = MYSQL_SELECT_SCHEME_IDS_FOR_ACTIONS;
+    private static final String MSSQL_SELECT_SCHEME_IDS_FOR_ACTIONS = MYSQL_SELECT_SCHEME_IDS_FOR_ACTIONS;
+    private static final String ORACLE_SELECT_SCHEME_IDS_FOR_ACTIONS = MYSQL_SELECT_SCHEME_IDS_FOR_ACTIONS;
+
+    // private static final String MYSQL_UPDATE_SCHEME_IDS_FOR_ACTIONS = "UPDATE workflow_action SET scheme_id = ?,requires_checkout_option = ?  WHERE id = ?";
+    private static final String MYSQL_UPDATE_SCHEME_IDS_FOR_ACTIONS = "UPDATE workflow_action SET scheme_id = ?  WHERE id = ?";
+    private static final String POSTGRES_UPDATE_SCHEME_IDS_FOR_ACTIONS = MYSQL_UPDATE_SCHEME_IDS_FOR_ACTIONS;
+    private static final String MSSQL_UPDATE_SCHEME_IDS_FOR_ACTIONS = MYSQL_UPDATE_SCHEME_IDS_FOR_ACTIONS;
+    private static final String ORACLE_UPDATE_SCHEME_IDS_FOR_ACTIONS = MYSQL_UPDATE_SCHEME_IDS_FOR_ACTIONS;
+
+    private static final String MYSQL_SELECT_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "SELECT index_name FROM information_schema.statistics WHERE table_name = 'workflow_scheme_x_structure' AND column_name = 'structure_id' AND index_name = 'workflow_idx_scheme_structure_2'";
+    private static final String POSTGRES_SELECT_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "SELECT indexname FROM pg_indexes WHERE tablename = 'workflow_scheme_x_structure' AND indexname = 'workflow_idx_scheme_structure_2'";
+    private static final String MSSQL_SELECT_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "SELECT name FROM sys.indexes WHERE object_id = (SELECT object_id FROM sys.objects WHERE name = 'workflow_scheme_x_structure') AND name = 'workflow_idx_scheme_structure_2'";
+    private static final String ORACLE_SELECT_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "SELECT index_name, table_owner, table_name, uniqueness FROM user_indexes WHERE table_name = 'WORKFLOW_SCHEME_X_STRUCTURE' AND index_name = 'WK_IDX_SCHEME_STR_2'";
+
+    private static final String MYSQL_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "DROP INDEX workflow_idx_scheme_structure_2 ON workflow_scheme_x_structure";
+    private static final String POSTGRES_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "DROP INDEX IF EXISTS workflow_idx_scheme_structure_2 CASCADE";
+    private static final String MSSQL_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = MYSQL_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX;
+    private static final String ORACLE_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX = "DROP INDEX WK_IDX_SCHEME_STR_2";
+
+    private static final String MYSQL_DROP_WORKFLOW_ACTION__STEP_FK = "alter table workflow_action drop foreign key fooconstraint     DROP INDEX workflow_idx_scheme_structure_2 ON workflow_scheme_x_structure";
+    private static final String POSTGRES_DROP_WORKFLOW_ACTION__STEP_FK = "DROP INDEX IF EXISTS workflow_idx_scheme_structure_2 CASCADE";
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use already-defined constant 'POSTGRES_DROP_WORKFLOW_SCHEME_X_STRUCTURE_INDEX' instead of duplicating its value here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00115)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "POSTGRES_DROP_WORKFLOW_ACTION__STEP_FK" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)